Source/santad/Santad.mm

```diff
@@ -504,6 +504,35 @@ void SantadMain(std::shared_ptr<EndpointSecurityAPI> esapi, std::shared_ptr<Logg
 
                                    tty_writer->EnableSilentTTYMode(newBool);
                                  }],
+    [[SNTKVOManager alloc] initWithObject:configurator
+                                 selector:@selector(enableMachineIDDecoration)
+                                     type:[NSNumber class]
+                                 callback:^(NSNumber *oldValue, NSNumber *newValue) {
+                                   BOOL oldBool = [oldValue boolValue];
+                                   BOOL newBool = [newValue boolValue];
+
+                                   if (oldBool == newBool) {
+                                     return;
+                                   }
+
+                                   LOGI(@"enableMachineIDDecoration changed %d -> %d", oldBool,
```

Comment:
```suggestion
                                   LOGD(@"EnableMachineIDDecoration changed %d -> %d", oldBool,
```

Reply:
If you feel strongly that this should be info instead of debug that's fine, but the key should be capitalized

Reply:
I went through and standardized casing for all KVO logs.